script/e2e-test.sh

```diff
@@ -189,7 +189,7 @@ installOrUpgradeKubeapps() {
     --set frontend.replicaCount=1
     --set kubeops.replicaCount=1
     --set dashboard.replicaCount=1
-    --set kubeappsapis.replicaCount=1
+    --set kubeappsapis.replicaCount=2
```

Comment:
Oh my... I didn't remember that scale down. I wondered how can it be possible if we already set two replicas and lowered the memlimits to 256!
Thanks for the fix!
(I wonder if the Bitnami people are also manually setting the num of replicas...)